composer.json

```diff
@@ -12,14 +12,13 @@
     "require": {
         "php": ">=7.1.0",
         "silverstripe/recipe-plugin": "^1",
-        "silverstripe/recipe-cms": "^4.4.1",
-        "silverstripe/admin": "1.4.2 as 1.4.1",
+        "silverstripe/recipe-cms": "^4.6",
         "silverstripe/bambusa-theme": "dev-master",
         "silverstripe/recipe-content-blocks": "^2.3",
```

Comment:
I'd just change all the other carets to major only e.g. `^2` for tidiness

Reply:
I think it can be useful to know what version was originally installed when the constraint was added.